src/setup.h

```diff
@@ -15,7 +17,7 @@ void
 uzbl_events_free ();
 
 void
-uzbl_gui_init ();
+uzbl_gui_init (WebKitWebContext *context);
```

Comment:
What thread will own the `UzblWeb` struct? Why not just put it in `UzblGui` or `UzblVariables`?